production.py

```diff
@@ -390,7 +390,7 @@ def calc_group_operation(self, kind):
             and x.kind == kind]
         res = {}
         for op in self.operation_costs:
-            if not op.operation or op.kind != kind:
+            if not op.operation or op.kind != kind or not op.quantity:
```

Comment:
Same as calc group.

Reply:
Idem as above, the divisor is `qty = op.quantity + op_cost.quantity`.

Reply:
Similar to calc group.

Reply:
It should not be given here, there is no need to check, because it already leaves the unit_price at 0 if the qty is 0.

Reply:
See changeset dcd7c13bb6b6fa886b3c8903d541afc440051882